### DISCUSSION TITLE
为什么新版的客户端有些视频播放不了

### DISCUSSION THREAD

**Discussion by Charry123453** [community/community#110677] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

升级到最新版客户端绝大部分视频点进去都无法播放了
![屏幕截图 2024-03-02 103331](https://github.com/community/community/assets/101203613/5def0b98-acd5-4de0-a90f-899f41afc8ff)

**Comment by SebastienXia** [comment]
Hi, @Charry123453 
I think you are not in the right place to ask this question here, I recognize the Bilibili background !!

**Comment by Akash1134** [comment]
Hi @Charry123453 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
